grammars/erlang.bnf

```diff
@@ -321,6 +324,34 @@ argument_definition_list ::= '(' argument_definition? (',' argument_definition)*
 clause_guard ::= when <<guarded guard>> {pin=1}
 clause_body ::= '->' exprs {pin=1}
 
+app_expression ::= '{' 'application' ',' module_ref ',' app_parameters '}' {elementType=tuple_expression}
+app_parameters ::= '[' app_parameter? (',' app_parameter)* ']' {elementType=list_expression}
+private is_app_hack ::= is_app 'xxx' //TODO remove when Grammar-Kit won't generate a lookahead for '{' in is_app code
```

Comment:
is_app_hack relates to is_app, not to everything else you have here. Wouldn't it be better to keep related things together?